apps/pie-storybook/.storybook/preview-head.html

```diff
@@ -1,3 +1,8 @@
 <script>
     window.global = window;
-</script>
\ No newline at end of file
+</script>
+<style>
+    body {
+        font-family: var(--dt-font-interactive-m-family);
```

Comment:
Not necessarily for this PR, but should we set this in `pie-css`? (I'm thinking yes, but would be good to get another opinion!)

Reply:
probably one of that epic! I agree!